scripts/state_backup.sh

```
@@ -1,30 +1,53 @@
 }
-abci_timestamp() { date +%Y%m%dT%H%M%S; }
+timestamp() { date +%Y%m%dT%H%M%S; }
 abci_resolve_config() {
   local cfg=${1:-}
-  if [ -n "$cfg" ]; then echo "$cfg"; else ${DASHMATE_BIN} config default; fi
+  if [ -n "$cfg" ]; then echo "$cfg"; else "${DASHMATE_CMD[@]}" config default; fi
 }
 abci_resolve_volume() {
   local cfg=$1
-  local project=$(${DASHMATE_BIN} config envs --config "$cfg" | awk -F= '$1=="COMPOSE_PROJECT_NAME"{print $2}')
+  local project=$("${DASHMATE_CMD[@]}" config envs --config "$cfg" | awk -F= '$1=="COMPOSE_PROJECT_NAME"{print $2}')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)